### ISSUE TITLE
Some English text on the French form

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3088]
**HC Assessment Label: Serious**

Pages: Programme Avancement de la recherche sur les normes d'accessibilité : Déclaration d’intérêt  
URL: https://forms-staging.cdssandbox.xyz/fr/id/134
User agent:  Chrome (v119.0.6045.200)
Language: French

Issue Description: 
Some English text on the French form. For details, see attached screenshot.

Steps to reproduce:

1. Open the above URL in the Chrome browser
2. Visual check of the contents on the pages

Expected result: All the content should be provided in the language of the page.

**Comment by srtalbot** [comment]
Result of testing on staging with an out of date form.
